src/library/components/InputRadio.tsx

```diff
@@ -91,9 +93,38 @@ const InputRadio = forwardRef(
       setHelpTextId('radioHelpText' + idNum);
     }, []);
 
+    useEffect(() => {
+      if (addonProps.buttonGroup) {
+        setIsGroupStyle(addonProps.buttonGroup);
+      }
+    }, [isGroupStyle, addonProps]);
+
+    let inputRadioStyles = {
+      containerClass: 'bsds-input',
+      radioContainerClass: 'bsds-input-radio',
+      inputRadioClass: 'bsds-input-radio-input',
+      labelClass: 'bsds-input-radio-label',
+      inputHelptext: 'bsds-input-help-text'
+    };
+
+    const inputButtonGroupStyles = {
+      containerClass: 'bsds-input-button-group',
+      radioContainerClass: 'bsds-input-radio-button-group',
+      inputRadioClass: 'bsds-input-radio-input',
+      labelClass: 'bsds-input-radio-label-button-group',
+      inputHelptext: 'bsds-input-help-text'
+    };
+
+    if (isGroupStyle) {
+      inputRadioStyles = { ...inputButtonGroupStyles };
+      if (inputUnavailable) {
+        inputRadioStyles.labelClass = 'bsds-input-radio-label-button-group-unavailable';
```

Comment:
i see what you mean about confusing class names 😬
let's chat about some naming options that could help shorten/clear them up